### PR TITLE
changed role title text to Default Allowed Roles

### PR DESCRIPTION
**Pull request by kuluruvineeth**
Fixes this issue: https://github.com/nhost/nhost/issues/1561

@szilarddoro Please review this pull request and let me know if still any changes are to be made, Thanks. Your further guidance and assistance would be really greatful for me to contribute more to this project.

### CODE REVIEW

**Review by szilarddoro — CHANGES_REQUESTED** [review]
This label should also be changed in [the documentation](https://github.com/nhost/nhost/blob/main/docs/docs/authentication/users.mdx).

**Comment by kuluruvineeth** [comment]
> This label should also be changed in [the documentation](https://github.com/nhost/nhost/blob/main/docs/docs/authentication/users.mdx).

@szilarddoro I have made changes in the documentation part too, thanks for reviewing it.

File: docs/docs/authentication/users.mdx [hunk 1]
```diff
@@ -34,7 +34,7 @@ The default role is used when no role is specified in the request. By default, u
 
 You can change what the default role for new users should be at **Settings -> Roles and Permissions**.
 
-### Allowed Roles
+### Default Allowed Roles
 
 Allowed roles are roles the user is allowed to use when making a request. Usually, you would change the role from `user` (the default role) to some other role because you want to use a different role to resolve permissions for a particular request.
```
Comment (szilarddoro) [inline docs/docs/authentication/users.mdx:39]: See my other comment. We need to change the text here too.

**Review by szilarddoro — CHANGES_REQUESTED** [review]

File: docs/docs/authentication/users.mdx (hunk 1 above)
Comment (kuluruvineeth) [reply docs/docs/authentication/users.mdx]: @

> See my other comment. We need to change the text here too.

@szilarddoro my apologies for misunderstanding, now i have changed the text in description part as well.

**Comment by dbarrosop** [comment]
Thanks for the contribution, we ended up fixing on the PR https://github.com/nhost/nhost/pull/2452

**Pull request closed without merging**